src/components/layout/Navbar/index.tsx

```diff
@@ -50,6 +81,34 @@ const Navbar = ({ user }: NavbarProps) => {
           </Link>
         </nav>
       </div>
+      {/* Mobile Menu Dropdown */}
+      <div className={`${styles.mobileNav} ${menuOpen && styles.open}`}>
+        <Link className={styles.mobileNavItem} onClick={() => setMenuOpen(false)} href="/">
+          <CalendarIcon color="var(--theme-text-on-background-1)" className={styles.iconLink} />
+          Events
+        </Link>
+        <Link
+          className={styles.mobileNavItem}
+          onClick={() => setMenuOpen(false)}
+          href="/leaderboard"
+        >
+          <LeaderboardIcon color="var(--theme-text-on-background-1)" className={styles.iconLink} />
```

Comment:
Is there a reason we're defining the color here instead of in the `.iconLink` class name?